src/mumble/PulseAudio.cpp

```diff
@@ -975,3 +1015,75 @@ void PulseAudioOutput::run() {
 		qwcWait.wait(&qmMutex);
 	qmMutex.unlock();
 }
+
+PulseAudio::PulseAudio() : m_ok(false) {
+	const QStringList alternatives{ QLatin1String("libpulse.so"), QLatin1String("libpulse.so.0") };
+
+	for (const QString &alternative : alternatives) {
+		m_lib.setFileName(alternative);
+		if (m_lib.load()) {
+			break;
+		}
+	}
+
+	if (!m_lib.isLoaded()) {
+		return;
+	}
```

Comment:
Why don't we have to perform a similarly thorough search for PulseAudio as we don for Opus?
https://github.com/mumble-voip/mumble/blob/bb925f40de90ac178b2ac50e2432e262e80ca428/src/mumble/OpusCodec.cpp#L31-L73

Is it because we usually install Opus with Mumble but don't do the same with PulseAudio?

Reply:
Also, our PulseAudio implementation is only tested (and built) on Linux.